docs/user/README.md

```diff
@@ -259,9 +259,11 @@ variables is provided here. If an environment variable is not set, the specified
 | **Variable**                                     | **Default** | **Description**                                                                                                                                                                                                                                                                                          |
 |--------------------------------------------------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | MODELARDBD_PORT                                  | 9999        | The port of the Apache Arrow Flight Server.                                                                                                                                                                                                                                                              |
+| MODELARDBD_IP_ADDRESS                            | 127.0.0.1   | The IP address of the Apache Arrow Flight Server.                                                                                                                                                                                                                                                        |
 | MODELARDBD_UNCOMPRESSED_RESERVED_MEMORY_IN_BYTES | 512 MB      | The amount of memory to reserve for storing uncompressed data buffers.                                                                                                                                                                                                                                   |
 | MODELARDBD_COMPRESSED_RESERVED_MEMORY_IN_BYTES   | 512 MB      | The amount of memory to reserve for storing compressed data buffers.                                                                                                                                                                                                                                     |
 | MODELARDBD_TRANSFER_BATCH_SIZE_IN_BYTES          | 64 MB       | The amount of data that must be collected before transferring a batch of data to the remote object store.                                                                                                                                                                                                |
+| MODELARDBD_TRANSFER_TIME_IN_SECONDS              | Disabled    | The number of seconds between each transfer of data to the remote object store.                                                                                                                                                                                                                          |
```

Comment:
If I understand it correctly, it is possible to set this value to 0 and then the system will always transfer data immediately. Is that correct? Should it be mentioned here?

Reply:
It is technically correct but I think it is worth looking into edge-cases like this for the configuration so I have created a [work item](https://dev.azure.com/ModelarData/ModelarDB%20Implementation/_workitems/edit/824) to look into this further.